Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rust:1.50
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN rustup toolchain install nightly && \
+    rustup default nightly
+
+RUN cargo build --release
```

Comment:
Shouldn't this be `cargo run --release`?

Reply:
No, `RUN` steps are evaluated at build time. This way, the image ships with a pre-built ready-to-run binary

Reply:
Aha, ok